ironfish/src/network/peers/peerManager.ts

```diff
@@ -83,6 +84,12 @@ export class PeerManager {
    */
   peers: Array<Peer> = []
 
+  /**
+   * PeerAddressManager allows the storage and persistence of addresses that can
+   * map to peers.
+   */
```

Comment:
This is comment is describing the type, not the instance here. I feel like this comment should be on the type itself instead of here.

Reply:
👍🏽